GoogleUtilities/FileDataStorage/GULStorageHeartbeat.m

```diff
@@ -35,20 +36,61 @@ - (instancetype)initWithFileURL:(NSURL *)fileURL {
   return self;
 }
 
+- (nullable NSMutableDictionary *)getDictionary {
```

Comment:
nit: A method name like `heartbeatDictionary` will be more Objective-C friendly.